.github/workflows/validate-workflows-files.yml

```diff
@@ -1,12 +1,23 @@
 name: Validate workflows files
-on: [ push, pull_request ]
+on:
+  push:
+    branches:
+     - main
+    paths:
+      - '.github/workflows/*.yml'
```

Comment:
I don't think we need to check it on push - I mean, we will not push anything to the `main` branch without a PR

Reply:
If we remove the branch restriction, then the workflow will run on branches in this repo twice.